test/goose/brokers/redis/batch_test.clj

```diff
@@ -48,4 +59,106 @@
       (redis-batch/enqueue tu/redis-conn batch)
 
       (is (= (redis-cmds/parse-map tu/redis-conn batch-state-key) batch-state))
-      (is (= (redis-cmds/set-members tu/redis-conn enqueued-job-set) job-ids)))))
```

Comment:
Always configure auto-format/IDE to leave a newline at end of file to avoid git-diffs like these.